super_maeket.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Super Market</title>
+    <link rel="stylesheet" href="super_market.css">
+</head>
+<body>
+    <nav>
+      <h1 class="H1_in_Nav">SUPER MARKET</h1> 
```

Comment:
Since you didn't use this class name `H1_in_Nav ` anywhere in your code, you consider removing unless you have a future use for it.

Reply:
Ok 👍